config/v1/tests/nodes.config.openshift.io/AAA_ungated.yaml

```diff
@@ -12,3 +12,56 @@ tests:
         apiVersion: config.openshift.io/v1
         kind: Node
         spec: {}
+    - name: Should be able to create a Node object with cgroupMode set to "v2"
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v2"
+    - name: Should not allow to create a Node object with the cgroupMode set to "v1"
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "v1"
+      expectedError: "cgroups v1 is not supported on openshift anymore"
+    - name: Should not allow to create a Node object with the cgroupMode set to some random value
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          cgroupMode: "unknown"
+      expectedError: "spec.cgroupMode: Unsupported value: \"unknown\""
+  onUpdate:
```

Comment:
You also need to add some ratcheting tests, where you remove the new validation, apply the initial object, and then show that you can update other fields with a persisted bad value. Please check the tests/readme for ratcheting to see how this works

Reply:
Added the ratcheting tests. Thanks for the info